packages/dropdown/test/dropdown.test.ts

```diff
@@ -457,13 +455,12 @@ describe('Dropdown', () => {
         expect(el.selectedItemText).to.equal('');
     });
 
-    it('resets value when item not available', async () => {
+    it('allows event listeners on child items', async () => {
         const mouseenterSpy = spy();
         const handleMouseenter = (): void => mouseenterSpy();
         const el = await fixture<Dropdown>(
             html`
                 <sp-dropdown
-                    value="missing"
```

Comment:
Double check this is a correct change to the test...

Reply:
Ah just checked the dropdown class. In the `updated` method, Dropdown clears it's value if the value doesn't match with a `Menu Item`.

Maybe we can leave these two lines in? or separate it into another test

Reply:
Good eye!

After taking a look at the full file, I remember why it's being removed, however.

The _actual_ `'resets value when item not available'`, one test above in the file, already tests that functionality, so this was clarifying the intent of the test in question.

So, removed and separated into another test, already!